Source/animationinfos.cpp

```diff
@@ -48,4 +48,78 @@ int AnimationInfos::GetFrameToUseForRendering()
 	return absoluteAnimationFrame;
 }
 
+void AnimationInfos::SetNewAnimation(uint8_t *Peq, int numFrames, int Delay, AnimationDistributionParams params /*= AnimationDistributionParams::None*/, int numSkippedFrames /*= 0*/, int distributeFramesBeforeFrame /*= 0*/)
```

Comment:
Parameter names should not be PascalCase I think?

Reply:
```suggestion
void AnimationInfos::SetNewAnimation(uint8_t *peq, int numFrames, int delay, animationDistributionParams params /*= AnimationDistributionParams::None*/, int numSkippedFrames /*= 0*/, int distributeFramesBeforeFrame /*= 0*/)
```

Reply:
Fixed.
Do you want to also adjust `NewPlrAnim`?
That was the original location.

Reply:
Yeah we are in the process of unifying the style used in the project, sorry for the current mess.

Reply:
No Problem. fixed 🙂